src/tomography/rules.mk

```diff
@@ -65,6 +65,24 @@ tomography_MODULES = \
 	$(FC_MODDIR)/tomography_model_iso.$(FC_MODEXT) \
 	$(EMPTY_MACRO)
 
+postprocess_TARGETS = \
```

Comment:
Oh, I see you added them separately. It would be simpler to do:
```
tomography_TARGETS += $(postprocess_TARGETS)
```
but it does work for building the way you have it.

However, it doesn't look like the `*_OBJECTS` are added also?  Then `make clean` from the directory would be incorrect.